test/optics/interference_minimum_test.py

```diff
@@ -0,0 +1,39 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to, prefixes,
+)
+from symplyphysics.laws.optics import interference_minimum as minimum_law
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    minimum_number = 2
+    wave_length = Quantity(550 * prefixes.nano * units.meters)
+    Args = namedtuple("Args", ["wave_length", "minimum_number"])
+    return Args(wave_length=wave_length, minimum_number=minimum_number)
+
+
+def test_basic_travel_difference(test_args):
+    result = minimum_law.calculate_travel_difference(test_args.wave_length, test_args.minimum_number)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.length)
+    result_travel_difference = convert_to(result, prefixes.nano * units.meters).evalf(5)
```

Comment:
CI is failing somewhere around this line. Try to use brackets `(prefixes.nano * units.meters)`

Reply:
It didn't work. And if we use only `units.meters`, this error occurred again.